AcademyResidentInformationApi/V1/UseCase/Interfaces/IGetResidentByIdUseCase.cs

```diff
@@ -0,0 +1,9 @@
+using AcademyResidentInformationApi.V1.Boundary.Responses;
+
+namespace AcademyResidentInformationApi.V1.UseCase.Interfaces
+{
+    public interface IGetResidentByIdUseCase
+    {
+        public ResidentInformation Execute(int id);
```

Comment:
You don't need the `public` accessor in interfaces as everything is public

Reply:
true, same for the interface declaration?

Reply:
I think you still need it there